util/util.go

```diff
@@ -40,9 +39,6 @@ func GetSecrets(clientSet kubernetes.Interface, namespace, name, key string) ([]
 		secret, err = secretsIf.Get(name, metav1.GetOptions{})
 		if err != nil {
 			log.Warnf("Failed to get secret '%s': %v", name, err)
-			if !errorsutil.IsTransientErr(err) {
```

Comment:
same here, might be used by code that does not want to retry these errors

Reply:
reverted.